src/components/skills-quiz/SkillsQuizStepper.jsx

```diff
@@ -157,154 +160,162 @@ const SkillsQuizStepper = () => {
   return (
     <>
       <Stepper activeKey={currentStep}>
-        <FullscreenModal
```

Comment:
Any reason to shift from `<FullscreenModal>` to `<ModalDialog>`?

Reply:
FullScreenModal is using ModalDialog behind the scenes and when we were using FUllScreenModal i was not being able to get rid of the greenish title bar containing Skills Quiz Heading, to do that customisation i chose to use the ModalDialog directly and i have tested, things are working fine.